requirements-travis.txt

```diff
@@ -11,7 +11,7 @@ PyYAML==3.11
 Pillow==2.2.1
 snakefood==1.4
 networkx==1.9.1
-pygraphviz==1.3rc2
+#pygraphviz==1.3rc2
```

Comment:
Oh.... I'm an idiot.
I've tried to perform check localy on FC24 via $ make check. And pip -r requirements-travis.txt failed on graphvis, so i've installed it via dnf and then 'make check' succeed. 
BUT. I forget to uncomment it back, so this change appeared in commit. CRAP!
In my defence, I can say that it was 3 a.m. so my brain can't communicate of Travis build and my temp modification. Sorry.
In order to avoid such things in future,  i've enabled Travis for my repo so next time I'll get an automatic check next time.